closed/OpenJ9.gmk

```diff
@@ -249,9 +249,9 @@ endif
 
 # Adjust CRIU Support enablement flags.
 ifeq (true,$(OPENJ9_ENABLE_CRIU_SUPPORT))
-  FEATURE_SED_SCRIPT += $(call SedEnable,opt_criuSupport)
+  SPEC_SED_SCRIPT += $(call SedEnable,opt_criuSupport)
 else # OPENJ9_ENABLE_CRIU_SUPPORT
-  FEATURE_SED_SCRIPT += $(call SedDisable,opt_criuSupport)
+  SPEC_SED_SCRIPT += $(call SedDisable,opt_criuSupport)
```

Comment:
This won't help because the file that needs to be modified is `f9.flags` which is neither `.feature` nor `.spec`.

Reply:
Do you mean this? https://github.com/eclipse-openj9/openj9/blob/master/buildspecs/j9.flags#L1705

Reply:
I guess I could leave it as is and add it to https://github.com/eclipse-openj9/openj9/blob/master/buildspecs/core.feature

Reply:
Yes, I think a default for the flag belongs in some `.feature` file, `core.feature` seems reasonable enough.